src/utils/index.js

```diff
@@ -1 +1,4 @@
 export { getVideos } from "./videosHandler/getVideos";
+export { getLikesHandler } from "./likesHandler/getLikesHandler";
+export { addToLikesHandler } from "./likesHandler/addToLikesHandles";
+export { removeFromLikesHandler } from "./likesHandler/removeFromLikesHandler";
```

Comment:
Similar to above scenario you could create an `index.js` file for all your utils and export these from there

Reply:
Same here as well.